concept/read_api_test.go

```diff
@@ -133,7 +140,7 @@ func TestGetConceptsByIDsUnmarshallingPayloadError(t *testing.T) {
 
 	ctx := tidUtils.TransactionAwareContext(context.Background(), tidUtils.NewTransactionID())
 	_, err := csAPI.GetConceptsByIDs(ctx, []string{"an-id"})
-	assert.EqualError(t, err, "invalid character '}' looking for beginning of value")
+	assert.Error(t, err)
```

Comment:
I think we can check for https://golang.org/pkg/encoding/json/#InvalidUnmarshalError
... or https://golang.org/pkg/encoding/json/#SyntaxError - I saw you use the last one in another place.

Reply:
I will fix it.